Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift

```diff
@@ -251,30 +251,26 @@ public enum PrimerPaymentMethodType: Codable, Equatable, Hashable {
                 .payNLGiropay,
                 .payNLIdeal,
                 .payNLPayconiq,
+                .primerTestKlarna,
+                .primerTestPayPal,
```

Comment:
I thought I put `.primerTestKlarna` and `.primerTestPaypal` in the right place the first time.
Shouldn’t they go at the bottom (`return true`) instead of being among the ones where we check the `Primer.shared.intent == .checkout` ?